docs/upgrade.html

```diff
@@ -21,6 +21,10 @@
 0.10.0.0 has <a href="#upgrade_10_breaking">potential breaking changes</a> (please review before upgrading) and
 there may be a <a href="#upgrade_10_performance_impact">performance impact during the upgrade</a>. Because new protocols
 are introduced, it is important to upgrade your Kafka clusters before upgrading your clients.
+<p/>
+<b>Notes to client with version 0.9.0.0: </b>Due to a bug introduced in 0.9.0.0,
+clients with ZooKeeper dependency may not be able to work with 0.10.0.0 broker. Therefore, 0.9.0.0 clients should
```

Comment:
@granthenke : Yes, we could keep version 2 in json. But the the drawback is the following. If we want to support upgrading form 0.9.0.0 to any future releases post 0.10.0, we can't bump the version in json forever. Given that (1) versioning the json in ZK is useful and (2) the issue in 0.9.0.0 is a bug and relatively few people are using 0.9.0.0 yet, I think it's probably better to change the ZK version now, but require people to upgrade from 0.9.0.1.